### PR TITLE
Double clicking a legend node opens symbol editor

### LINKED ISSUE #709: [FEATURE] Double clicking a legend node opens symbol editor directly
Original commit: https://github.com/qgis/QGIS/commit/c7c5c007be90076ba023090152410e2294bd402b by nyalldawson

Unfortunately this lazy coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by yjacolin**
Fix #709

[Doc contribution funded by camptocamp]

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -161,6 +161,13 @@ tools that were designed for the different kinds of vector data.
 
 .. ToDo: add information about the export options
 
+.. tip:: **Change the features display from legend**
+   
+   You can open the symbol editor dialog by double-clicking on the legend. If you
+   do a right-clic on the legend a contextual menu allow user to change the color, 
+   open the :menuselection`Edit symbol ...` or show/hide all features in this
+   class.
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: Actually it's not the `features of the class` that are shown/hidden but all the classes of the layer that are toggled visible or not (ie, all the features of the layer).
Note that there's also a PR that covers the contextual menu of vector layer ( #1142 )

Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: could you add the double-click info on your PR and we close this one?

Comment (yjacolin) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: review your PR, it is already done.

**Comment by DelazJ** [comment]
Yes but it's not at the same place so I don't think that we shouldn't keep yours. It's a tip so worth keeping imo (though I think we'll need in the future to review the list of tips in that chapter :) ).

**Comment by yjacolin** [comment]
I though exactly the same :)

**Pull request closed without merging**